pkg/server/io/container_io.go

```diff
@@ -71,6 +72,21 @@ func WithNewFIFOs(root string, tty, stdin bool) ContainerIOOpts {
 	}
 }
 
+// WithBinaryFIFOs specifies binary fifos for the container io
+func WithBinaryFIFOs(path string, tty bool) ContainerIOOpts {
```

Comment:
Does a tty even have a purpose for this work?

Reply:
good question. I didn't want to break the `tty` behavior initially, but this probably needs a different handling anyways (when `tty` is actually set). I'll probably move `WithBinaryFIFOs` to either `helpers_windows` or `container_io_windows` and get rid `tty`. upstream has different modes when binary is enabled with tty vs just binary and if we have a need for this in the future, we might need to change something in the shim.